features/flags.json

```diff
@@ -160,9 +160,9 @@
       "lastEditedAt": "2022-05-25T01:11:55.121Z"
     },
     "testnet_NEARORG": {
-      "enabled": false,
+      "enabled": true,
```

Comment:
This is flagged on temporarily for testing on the preview. Should be flagged off before merge

Reply:
This should really be a draft PR then to minimize the chance of forgetting to reset this. Alternatively create a branch off of this and create a draft PR for that, or better yet test it locally against testnet 🙂

Reply:
Tested both flag states in local but when choosing which one to be more public about going with flag on state to have all folks incl non technical be able to test the feature. Created #2799 for testing and reset this 👍